### PR TITLE
Prepare release 1.10.1

### PR DESCRIPTION
**Pull request by maxulysse**
nf-core bump-version 1.10.1

### CODE REVIEW

**Comment by drpatelh** [comment]
Need to update the CHANGELOG here too.

**Comment by maxulysse** [comment]
> Need to update the CHANGELOG here too and thank everyone for their contributions!

I'm just including the 3 of us since this is only TEMPLATE update and small refactoring.
I'll include the whole team for the nf-test POC PR

**Comment by maxulysse** [comment]
Where are the tests?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,23 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev - [date]
+## [[1.10.1](https://github.com/nf-core/fetchngs/releases/tag/1.10.1)] - 2023-09-25
```
Comment (drpatelh) [inline CHANGELOG.md:6]:
```suggestion
## [[1.10.1](https://github.com/nf-core/fetchngs/releases/tag/1.10.1)] - 2023-10-09
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,23 @@
...
+
+- [Adam Talbot](https://github.com/adamrtalbot)
+- [Harshil Patel](https://github.com/drpatelh)
+- [Maxime Garcia](https://github.com/maxulysse)
+
+Thank you to everyone else that has contributed by reporting bugs, enhancements or in any other way, shape or form.
+
+### Enhancements & fixes
+
+- [#177](https://github.com/nf-core/fetchngs/pull/177) - Template update with nf-core/tools 2.9
```
Comment (drpatelh) [inline CHANGELOG.md:20]:
```suggestion
- [#208](https://github.com/nf-core/fetchngs/pull/208) - Template update with nf-core/tools 2.10
```

**Review by drpatelh — APPROVED** [review]
Thanks!

**Pull request merged**